### PR TITLE
Fix analytics-netmanager merge styling  issue

### PR DESCRIPTION
**Pull request by Priscilla-AL**
What does this PR do?
Fixes analytics-netmanager merge styling  issues

How do I test it out?
cd netmanager
npm install
npm run prod-pc
checking out the appearance of the dashboard. Map, graphs, cards

### CODE REVIEW

**Review by Baalmart — CHANGES_REQUESTED** [review]
I think this brings back the old "locate" folder we used to used for the product...did you create this branch from the latest "staging"?
I also notice many changed files (5000+)
I think we need to do some investigation here.

**Review by Baalmart — COMMENTED** [review]
The work on the styling has also greatly improved as seen in the screenshot, thanks.
image

Also realised the lower 2 graphs are no longer loading content, not sure what the cause might be. Perhaps the lower screenshots will shed some light:
image

image

**Comment by Priscilla-AL** [comment]
> I think this brings back the old "locate" folder we used to used for the product...did you create this branch from the latest "staging"?
> I also notice many changed files (5000+)
> I think we need to do some investigation here.

Resolved

**Comment by Priscilla-AL** [comment]
> The work on the styling has also greatly improved as seen in the screenshot, thanks.
> image
> 
> Also realised the lower 2 graphs are no longer loading content, not sure what the cause might be. Perhaps the lower screenshots will shed some light:
> image
> 
> image

**Comment by Priscilla-AL** [comment]
> The work on the styling has also greatly improved as seen in the screenshot, thanks.
> image
> 
> Also realised the lower 2 graphs are no longer loading content, not sure what the cause might be. Perhaps the lower screenshots will shed some light:
> image
> 
> image

Must have had something to do with the 5000 :) files, I have reverted the commit.

File: netmanager/src/config/constants.js [hunk 1]
```diff
@@ -64,8 +64,6 @@ const prodConfig = {
     "http://34.78.78.202:30006/api/v1/monitor/network/devices/worstperforming",
   ADD_COMPONENT_URI:
     "http://34.78.78.202:30002/api/v1/devices/add/components?device=",
-  GET_ONLINE_OFFLINE_MAINTENANCE_STATUS:
```
Comment (Baalmart) [inline netmanager/src/config/constants.js:67]: I think we need to bring these constants back.

File: netmanager/src/config/constants.js [hunk 2]
```diff
@@ -195,8 +193,6 @@ const devConfig = {
   DEVICE_MAINTENANCE_LOG_URI:
     "http://localhost:4001/api/v1/device/monitor/maintenance_logs/",
   //DELETE_DEVICE_URI: "http://127.0.0.1:3000/api/api/v1/devices/ts/delete?device="
-  GET_ONLINE_OFFLINE_MAINTENANCE_STATUS:
```
Comment (Baalmart) [inline netmanager/src/config/constants.js:198]: Same here, we need to bring these constants back and just change environments using appropriate runtime commands (`npm run pc-dev`, etc) depending on your machine and the environment you would like to carry out tests in.

File: netmanager/src/views/pages/Dashboard/Dashboard.js [hunk 3]
```diff
@@ -525,62 +530,10 @@ const Dashboard = (props) => {
 
         <Grid item lg={6} md={6} sm={12} xl={6} xs={12}>
           <Grid item lg={12} sm={12} xl={12} xs={12}>
-            <Map id="rootMapContainerId" />
+            <div className={classes.mapContainer}>
+              <Map/>
+            </div>
           </Grid>
-
-          {/* <Grid container spacing={0} className="MapCardContent">
```
Comment (Baalmart) [inline netmanager/src/views/pages/Dashboard/Dashboard.js:531]: Just requesting @sserurich to check what this big chunk code removal means.

File: .gitignore [hunk 4]
```diff
@@ -1 +1 @@
-.vscode
```
Comment (Baalmart) [inline .gitignore:1]: I think we still need to maintain our .gitignore or just add new items. lets leave .vscode there.

**Review by Baalmart — CHANGES_REQUESTED** [review]

File: netmanager/src/config/constants.js (hunk 2 above)
Comment (Priscilla-AL) [reply netmanager/src/config/constants.js:198]: this is already in staging.

Comment (Baalmart) [reply netmanager/src/config/constants.js:198]: True, you are right. This is already in staging but this PR wants to remove them.

**Pull request closed without merging**